src/platform/src/common/components/Toast/CustomToast.jsx

```
@@ -1,73 +1,73 @@
+      className = 'bg-orange-600 text-white';
+      break;
+    case 'error':
+      icon = <MdError size={20} className="text-white" />;
+      className = 'bg-red-600 text-white';
+      break;
+    case 'info':
+      icon = <IoInformationCircle size={20} className="text-white" />;
+      className =
+        'bg-[var(--org-primary,var(--color-primary,#145fff))] text-white';
+      break;
+    case 'success':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)